metadata-ingestion/src/datahub/emitter/mcp_builder.py

```diff
@@ -125,6 +126,17 @@ class BucketKey(ContainerKey):
     bucket_name: str
 
 
+class NotebookKey(DatahubKey):
+    notebook_id: int
+    platform: str
+    instance: Optional[str]
+
+    def as_urn(self) -> str:
+        return make_dataset_urn_with_platform_instance(
+            platform=self.platform, platform_instance=self.instance, name=self.guid()
+        )
```

Comment:
This is kinda awkward but notebooks don't have a container structure (they have a folder path, but that can change)